Bender.yml

```diff
@@ -0,0 +1,22 @@
+package:
+  name: apb_adv_timer
+  authors: 
+    - "Florent Rotenberg <florent.rotenberg@greenwaves-technologies.com>"
+    - "Yao Zhang <yao.zhang@greenwaves-technologies.com>"
+    - "Antonio Pullini <pullinia@iis.ee.ethz.ch>"
+    - "Davide Schiavone <pschiavo@iis.ee.ethz.ch>"
+
```

Comment:
There is a missing dependency: [`apb_adv_timer` instantiates `pulp_clock_gating`](https://github.com/pulp-platform/apb_adv_timer/blob/0821c4c0ae200378828e311a2c0f1a22c7c8a77a/rtl/apb_adv_timer.sv#L451), which is not provided by this repository.